dashboard/app/models/script_level.rb

```diff
@@ -319,7 +319,7 @@ def summarize(include_prev_next = true, for_edit: false, user_id: nil)
       end
 
       summary = {
-        id: id,
+        id: id.to_s,
```

Comment:
Updated this to a string since I noticed that there was a dedicated effort some time back to make sure all level IDs were passed to the client as strings. We added this ID for lab2 purposes more recently, but because it wasn't a string, there was some inconsistency in how it was handled on the client. Changed to a string just to be consistent and safer.